kokoro/ubuntu/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+# create a timestamp file
+date +%s > timestamp.txt
+
+cd git/kokoro-codelab-elharo
```

Comment:
Yes, I'm trying to bootstrap this and make sure it works, whihc is tricky since the configs are split across two repos; but I've sent a cl to create a non-codelab configuration on the internal side. Meanwhile I'll update this.